dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "backstage-community-plugin-redhat-argocd",
+  "version": "1.7.0",
```

the dependancy used here is "@backstage-community/plugin-redhat-argocd": "1.6.7" so can we keep that as is?

```suggestion
  "version": "1.6.7",
```